kolibri/content/management/commands/importcontent.py

```diff
@@ -183,6 +183,10 @@ def _start_file_transfer(self, f, filetransfer, overall_progress_update):
             * False, 0 - the transfer fails and needs to retry.
         """
         try:
+            # Save the current progress value
+            original_value = self.progresstrackers[0].progress
```

Comment:
Looks like this list can potentially be empty? At least according to the failing test.

Reply:
ah thank you for pointing it out! Sorry I should have noticed it earlier.